src/layout/Table.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { TableCSS } from "../css";
+
+const tableStyle: TableCSS = {
+    borderSpacing: 0,
+    borderCollapse: "collapse",
+    width: "100%"
+};
+
+export const Table: React.FC<{ children: React.ReactNode }> = ({
```

Comment:
We could possibly improve this, depending on how often we think we'll want it, by passing in a `style` prop and merging that with the `tableStyle` styles?